packages/experimental/Avatar/src/AvatarTokens.ts

```diff
@@ -0,0 +1,24 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { AvatarTokens } from '.';
+import { convertCoinColorFluent } from './Avatar.helpers';
+
+export const AvatarStates: (keyof AvatarTokens)[] = [
```

Comment:
I don't think we need these states. The states are used to look up whether a certain subset of passed in default tokens is supposed to be applied. For example in the button, we have a subset of default tokens which we want to use when we have the primary state on the button. The Avatar component seems to not need any states, since we don't change what default tokens get applied based on the Avatar's current props.

Reply:
yes, you're right. I just took the code from PersonaCoin as it is. I'll work on new props/tokens for Avatar when I finish the spec